backend/src/conditions/rc.service.ts

```diff
@@ -46,7 +46,7 @@ export class RCService {
           db.raw('ST_MakePoint(longitude,latitude) as section_geom'),
           'timestamp',
         )
-        .where('type_index', '<', MeasurementType['DI'])
+        .where('type_index', '<=', MeasurementType['DI'])
         .whereNotNull('latitude')
         .whereNotNull('longitude');
```

Comment:
Shouldn't same be done for IRI?

Reply:
The issue with IRI is not of the same nature: there is no IRI data in our database (I will ask Magnus to fix the Lira importation script and redo the import). Regarding DI the issue was that i was excluding the DI data from being selected `MeasurementType['DI']` is just `22` (see enum below).

```typescript
export enum MeasurementType {
  Crack = 0,
  Rutting = 1,
  MacroTexture = 2,
  LaneMarking = 3,
  RumbleStrip = 4,
  Potholes = 5,
  DropOffCurb = 6,
  Joint = 7,
  Raveling = 8,
  Roughness = 9,
  RoadGeometry = 10,
  WaterEntrapment = 11,
  Shoving = 12,
  PickOut = 13,
  Bleeding = 14,
  SealedCrack = 15,
  Manholes = 16,
  Patch = 17,
  Pumping = 18,
  E_norm = 19,
  KPI = 20,
  Mu = 21,
  DI = 22,
  E_whl = 23,
  E_areo = 24,
  E_whl_std = 25,
  E_tire_std = 26,
  E_tire = 27,
  E_ineratia_slope = 28,
  E_inertia_slope_std = 29,
  E_areo_std = 30,
  E_norm_std = 31,
  mu_std = 32,
}
```

Reply:
Ahhh ok !